exchanges/bybit/bybit_websocket.go

```diff
@@ -592,195 +592,87 @@ func (by *Bybit) wsProcessKline(assetType asset.Item, resp *WebsocketResponse, t
 }
 
 func (by *Bybit) wsProcessPublicTicker(assetType asset.Item, resp *WebsocketResponse) error {
-	format, err := by.GetPairFormat(assetType, false)
+	tickResp := new(TickerItem)
+	if err := json.Unmarshal(resp.Data, tickResp); err != nil {
+		return err
+	}
+
+	p, err := currency.NewPairFromString(tickResp.Symbol)
```

Comment:
I think you should be using the matcher functions as incoming data does not contain delimiters. Take the following example:
As a regarded trader, I have `DOGE_USDT` enabled. `NewPairFromString` converts `DOGEUSDT` into `DOG`, `EUSDT` 
Things still save, but you will not be able to retreive the data from RPC endpoints for example

Surprisingly, knowledge of this behaviour was in the removed function `ExtractCurrencyPair`, but it was not used in actual code 🤷 I understand that you did not do the initial implementation, but may as well update it now to a more appropriate function

Reply:
> As a regarded trader, I have DOGE_USDT enabled

😂 😭 

> but it was not used in actual code

Also 😂 😭

Yep, on it. I think I skipped running this properly, which is a really bad oversight tbh.

Reply:
Fixed 29f2a5d and 720061c
Also had a few other fixes alongside, as well as fixing up the matcher being broken, so I pulled that in from #1534

Reply:
Is this now dependent on 1534?

Reply:
Nope, Not dependent.
I pulled one commit from it, but it doesn't matter which PR that commit ends up in.

Reply:
Just spotted the issue on that.
I had a DO NOT COMMIT marker on the commented section. Dunno where it went but that should have stopped me making that mistake of commenting out the DisableWebsocket calls 🤦 Maybe I need a DO NOT REMOVE marker above the DO NOT COMMIT marker 😂 

Kucoin fixed 79216e0
Linter fixed 7dc2f14